lib/server/monalisa-synchronization/MonAlisaClient.js

```diff
@@ -17,7 +17,7 @@ const https = require('https');
 const { extractLhcPeriod } = require('../utilities/extractLhcPeriod');
 const fs = require('fs').promises;
 
-const VALID_DATA_PASS_NAME_REGEX = /^LHC\d\d[a-zA-Z]+_[a-z]pass[^\s]*$/;
+const VALID_DATA_PASS_NAME_REGEX = /^LHC\d\d[a-zA-Z]+_([a-z]pass|skimming|skimmed)[^\s]*$/;
```

Comment:
Is `skimming/skimmed` considered a data pass as well or a different type of data? If it is the latter please update the name of the const

Reply:
it is data pass